test/service/test_manifest.py

```diff
@@ -171,6 +175,26 @@ def run(self, result: Optional[unittest.result.TestResult] = None) -> Optional[u
             m.return_value = False
             return super().run(result)
 
+    @manifest_test
+    def test_pfb_manifest(self):
+        self.maxDiff = None
+        bundle_fqid = self.bundle_fqid(uuid='587d74b4-1075-4bbf-b96a-4d1ede0481b2',
+                                       version='2018-10-10T022343.182000Z')
+        self._index_canned_bundle(bundle_fqid)
+        # We write entities differently depending on debug so we test both cases
+        for debug in (1, 0):
+            with self.subTest(debug=debug):
+                with mock.patch.object(type(config), 'debug', debug):
+                    response = self._get_manifest(ManifestFormat.terra_pfb, {})
+                    self.assertEqual(200, response.status_code)
+                    pfb_file = BytesIO(response.content)
+                    reader = fastavro.reader(pfb_file)
+                    records = list(reader)
+                    results_file = Path(__file__).parent / 'data' / 'pfb_manifest.results.json'
+                    with open(results_file, 'r') as f:
+                        expected_records = json.load(f)
+                    self.assertElasticsearchResultsEqual(records, expected_records)
```

Comment:
I wouldn't use assertElasticsearchResultsEqual here but instead freeze, sort and thaw `records`. Combined with the fact that the aforementioned process for creating the file yields `expected_records` already sorted, you should be able to just assert equality with `assertEqual` and get an error message with only significant changes.